packages/web-components/fast-components/src/tabs/tabs.styles.ts

```diff
@@ -81,11 +88,17 @@ export const TabsStyles = css`
     :host(.vertical) .activeIndicator {
         grid-column: 1;
         grid-row: 1;
-        width: 3px;
-        height: 20px;
+        width: 5px;
+        height: 100%;
+        margin-inline-end: 10px;
         border-radius: calc(var(--corner-radius) * 1px);
         align-self: center;
         background: var(--accent-fill-rest);
+        margin-top: 0;
+        border-top-left-radius: 0;
```

Comment:
These can be combined with the shorthand
```suggestion
        border-radius: calc(var(--corner-radius) * 1px) calc(var(--corner-radius) * 1px) 0 0;
```